src/BitStringAddresses/occupationnumberfs.jl

```diff
@@ -10,6 +10,8 @@ particles is runtime data, and can be retrieved with `num_particles(address)`.
 - `OccupationNumberFS{[M,T]}(onr)`: Construct from collection `onr` with `M` occupation
   numbers with type `T`. If unspecified, the type `T` of the occupation numbers is inferred
   from the type of the arguments.
+- `OccupationNumberFS{M[,T]}(onr)`: Construct a vacuum state with `M` modes. If `T` is
```

Comment:
```suggestion
- `OccupationNumberFS{M[,T]}()`: Construct a vacuum state with `M` modes. If `T` is
```
If I understand this correctly, `onr` shouldn't be passed in this case.

Reply:
Indeed. Thanks for spotting!